src/Surfnet/MessageBirdApiClientBundle/Tests/Service/MessagingServiceTest.php

```diff
@@ -83,4 +117,12 @@ private function expectStringContains($contains)
             return strpos($string, $contains) !== false;
         });
     }
+
+    protected function tearDown(): void
+    {
+        m::close();
+        parent::tearDown();
+    }
```

Comment:
This works great for adding the mockery expectations to the list of php unit assertion-count. Did you know you can achieve the same goal by adding this `trait` to your test class?

```
    use \Mockery\Adapter\Phpunit\MockeryPHPUnitIntegration;
```

See: `https://docs.mockery.io/en/latest/reference/phpunit_integration.html`

Reply:
As I not often (as in never) used mockery I did not know about this